python/GafferUI/ColorChooser.py

```diff
@@ -191,6 +191,17 @@ def getSwatchesVisible( self ) :
 
 		return self.__swatchRow.getVisible()
 
+	def setErrored( self, errored ) :
+
+		for n in self.__numericWidgets.values():
+			n.setErrored( errored )
+
+	def getErrored( self ) :
+		errored = false
```

Comment:
`NameError: name 'false' is not defined`

Could just be `any( w.getErrored() for w in self.__numericWidgets.values() )`.